dags/run_all_processors.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+from airflow.models import DAG
+from airflow.operators.latest_only_operator import LatestOnlyOperator
+import utils.helpers as helpers
```

Comment:
Hmm, Codacy does that, not sure if out of the box or you have to tick some boxes (rule boxes) to do it, but if we decided it's a keeper already, then I can transfer it to one of the umbrella emails we have and use it org-wide.